src/main/java/com/enonic/xp/app/welcome/WelcomePageScriptBean.java

```diff
@@ -202,19 +215,101 @@ public Object getSites()
     public Object getProjects()
     {
         List<ProjectJson> projects = new ArrayList<>();
-        projects.addAll( createAdminContext( ContentConstants.CONTENT_REPO_ID ).
-            callWith( () -> projectServiceSupplier.get().list() ).
-            stream().
-            sorted( Comparator.comparing( project -> project.getName().getRepoId().toString() ) ).
-            map( project -> ProjectJson.create().
-                project( project ).
-                iconAsBase64( getProjectIconAsBase64( project ) ).
-                build() ).
-            collect( Collectors.toList() ) );
+        projects.addAll(
+            createAdminContext( ContentConstants.CONTENT_REPO_ID ).callWith( () -> projectServiceSupplier.get().list() ).stream().sorted(
+                Comparator.comparing( project -> project.getName().getRepoId().toString() ) ).map(
+                project -> ProjectJson.create().project( project ).iconAsBase64( getProjectIconAsBase64( project ) ).build() ).collect(
+                Collectors.toList() ) );
 
         return new ProjectsMapper( projects );
     }
 
+    public Object getTemplateApplications()
+    {
+        List<TemplateApplicationJson> templateApps = new ArrayList<>();
+
+        Path filePath = HomeDir.get().toPath().resolve( "config" ).resolve( ".template" );
```

Comment:
`.template` is a hidden file. this may pose some "security" issues on Linux and Mac when it won't be obvious why XP SDK suddenly installs applications. (the only way to find out would be to call `ls -la` in config directory. 

`enonic-exp.template` is a more explicit and visible file

Reply:
what is "exp"?